client/src/pages/SearchPage/SearchPage.js

```diff
@@ -1,43 +1,41 @@
 import PropTypes from 'prop-types'
 import { useState } from 'react'
 import styled from 'styled-components/macro'
-import { cities } from '../../capstone.json'
 import AttractionCard from '../../components/AttractionCard/AttractionCard'
-import Header from '../../components/Header/Header'
 import Overlay from '../../components/Overlay/Overlay'
 import Searchbar from '../../components/Searchbar/Searchbar'
 import ScrollToTop from '../../services/ScrollToTop'
 
-export default function SearchPage({ handleAddLike, likedPlaces, open }) {
+export default function SearchPage({
+  handleAddLike,
+  likedPlaces,
+  open,
+  sights,
+}) {
   const [userInput, setUserInput] = useState('')
   return (
     <PageLayout>
       <ScrollToTop />
-      <Header />
       {open === true && <Overlay />}
       <Heading>Search the world</Heading>
       <Searchbar
         userInput={userInput}
         setUserInput={setUserInput}
         text={'Wanderlust?'}
       />
-      {cities.map(({ attraction, id }) => (
-        <CardLayout key={id}>
-          {attraction
-            .filter(item =>
-              item.name.toLowerCase().includes(userInput.toLowerCase().trim())
-            )
-            .map(({ name, image }) => (
-              <AttractionCard
-                key={name}
-                name={name}
-                image={image}
-                onAddLike={handleAddLike}
-                likedPlaces={likedPlaces}
-              />
-            ))}
-        </CardLayout>
-      ))}
+      {sights
+        .filter(sight =>
```

Comment:
I noticed in a few components that you do the `filter` in the return statement. It would be nicer to do it before the return statement and save it as a variable.

Reply:
Thank you! 🚀